### PR TITLE
Feature/lti bundle recipe

### PR DESCRIPTION
**Pull request by ekkinox**
| Q             | A
| ------------- | ---
| License       | MIT

- Added default config files
- Added instructions for development keys generation

### CODE REVIEW

**Comment by ekkinox** [comment]
@Nyholm hello :)

Can it me merged since this bundle does not aim to support SF 3.4 ? see https://travis-ci.org/github/symfony/recipes-contrib/builds/688304131?utm_source=github_status&utm_medium=notification

File: oat-sa/bundle-lti1p3/2.0/config/secrets/dev/private.key [hunk 1]
```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```
Comment (Nyholm) [inline oat-sa/bundle-lti1p3/2.0/config/secrets/dev/private.key:1]: I dont think it is a good idea to have a private key in the recipe. 
I understand it is just for debugging but the term "private" do have a meaning. =)

Comment (ekkinox) [reply oat-sa/bundle-lti1p3/2.0/config/secrets/dev/private.key:1]: true :) its just a key pair to make the bundle ready out of the box, only for dev env, with a warning very explicit on bundle install to not use them in prod.

I see 2 options:

Comment (Nyholm) [reply oat-sa/bundle-lti1p3/2.0/config/secrets/dev/private.key:1]: I think the latter is better. 

The former will make the key unusable.

Comment (ekkinox) [reply oat-sa/bundle-lti1p3/2.0/config/secrets/dev/private.key:1]: yes indeed, fixed :)

**Pull request merged**